src/main.js

```diff
@@ -227,7 +227,12 @@ module.exports = {
   async fixJob(isSave = false) {
     const textEditor = atom.workspace.getActiveTextEditor()
 
-    if (!textEditor || textEditor.isModified()) {
+    if (!textEditor || !atom.workspace.isTextEditor(textEditor)) {
+      // Silently return if the TextEditor is invalid
+      return
```

Comment:
The issue I see with that is the initially reported scenario: Making changes to dozens (if not hundreds) of files at once. If that's what was happening your console would suddenly be spammed with messages with no way to determine what went wrong, or how to fix it (which you couldn't anyway here).